src/utils/fs.rs

```diff
@@ -16,6 +16,27 @@ pub fn file_to_string<P: AsRef<Path>>(path: P) -> Result<String> {
     Ok(content)
 }
 
+/// Changes path to use standard UNIX/URL separators
+pub fn normalize_path(path: &str) -> String {
```

Comment:
This is just a thought, but would it make sense to use a `Path` or `Url` here to make it more obvious what these strings are?

Reply:
The function operates on a string but `Path` needs to be converted into a string, which can throw an error. Both places this function is used already converted the path to a string. I don't think it's worth the trouble.